clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/deploy/converters/securitygroup/UpsertOpenstackSecurityGroupAtomicOperationConverter.groovy

```diff
@@ -26,7 +26,7 @@ import com.netflix.spinnaker.clouddriver.security.AbstractAtomicOperationsCreden
 import org.springframework.stereotype.Component
 
 @OpenstackOperation(AtomicOperations.UPSERT_SECURITY_GROUP)
-@Component("upsertSecurityGroupDescription")
+@Component("upsertOpenstackSecurityGroupDescription")
```

Comment:
Removed for all similar ones in this PR. Will leave other refactoring to new PR.